api/uninstall/uninstall_test.go

```diff
@@ -100,6 +189,26 @@ func (s *UninstallTestSuite) TestShouldReturnNotFoundErrorIfItHasUnavailableRele
 	s.mockService.AssertExpectations(s.T())
 }
 
+func (s *UninstallTestSuite) TestShouldReturnNotFoundErrorIfItHasUnavailableReleaseNameRestful() {
+	unavailableReleaseName := "unknown_release"
```

Comment:
let's not have duplicates for all handler, we can change the existing handler to fit additional docs requirements